### PR TITLE
Update BUILDING-FEDORA.md

### PR DESCRIPTION
**Pull request by T0MASD**
update url to new org, use shallow clone, save x3 clone time and size

```
$ time git clone https://github.com/royshil/obs-backgroundremoval.git
Cloning into 'obs-backgroundremoval'...
remote: Enumerating objects: 2081, done.
remote: Counting objects: 100% (1204/1204), done.
remote: Compressing objects: 100% (669/669), done.
remote: Total 2081 (delta 750), reused 847 (delta 521), pack-reused 877
Receiving objects: 100% (2081/2081), 56.39 MiB | 1.85 MiB/s, done.
Resolving deltas: 100% (1119/1119), done.

real	0m31.949s
user	0m2.081s
sys	0m1.675s

```

vs
```
$ time git clone https://github.com/royshil/obs-backgroundremoval.git --depth=1
Cloning into 'obs-backgroundremoval'...
remote: Enumerating objects: 257, done.
remote: Counting objects: 100% (257/257), done.
remote: Compressing objects: 100% (219/219), done.
remote: Total 257 (delta 25), reused 152 (delta 21), pack-reused 0
Receiving objects: 100% (257/257), 24.17 MiB | 2.33 MiB/s, done.
Resolving deltas: 100% (25/25), done.

real	0m11.341s
user	0m0.779s
sys	0m0.669s
```

### CODE REVIEW

**Review by royshil — APPROVED** [review]
LGTM

File: docs/BUILDING-FEDORA.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ sudo dnf install cmake gcc-c++ ninja-build obs-studio-devel opencv-devel qt6-qtb
 
 Clone the repository and set up the submodules:
 ```
-git clone https://github.com/royshil/obs-backgroundremoval.git
+git clone https://github.com/obs-ai/obs-backgroundremoval.git --depth=1
```
Comment (umireon) [inline docs/BUILDING-FEDORA.md]:
```suggestion
git clone https://github.com/obs-ai/obs-backgroundremoval.git
```

**Review by umireon — CHANGES_REQUESTED** [review]

**Comment by umireon** [comment]
I don't think we should recommend depth=1. This will be problematic.

**Comment by umireon** [comment]
Saving 20 secs does not worth specifying --depth=1.

**Comment by royshil** [comment]
@umireon it's just for the build, this is in the readme, we can comment and say that it will get only the master branch without any history and that for actual development you need to `git fetch --unshallow`

**Comment by umireon** [comment]
If we suggest users use the full history, all the users can build our plugin correctly and some users can use git commands without additional operation when they need to use source codes on PR.
The `--depth=1` option will require us the additional context to be added to the README and the additional context will confuse users.

IMO `--depth=1` is a very harmful option with almost no benefit. AFAIK the homebrew project had stopped using --depth=1 because it was very harmful.

**Comment by umireon** [comment]
I don't want to ask Fedora users who have a build problem if their git copy is shallow or deep because this question is hard to answer.

**Comment by royshil** [comment]
okay i see your point.
we can keep it as is for now and see if anyone has complaints in the future about the clone time

**Comment by umireon** [comment]
I will agree with --depth=1 when cloning our repository takes 30 mins, of course!

**Review by umireon — APPROVED** [review]
LGTM

**Pull request merged**